src/main/java/org/marek/model/User.java

```diff
@@ -0,0 +1,18 @@
+package org.marek.model;
+
+import javax.persistence.Entity;
+
+@Entity
+
+public class User {
+    private Integer id;
```

Comment:
Tohle už je v těch materiálech, které jsem ti poslal. ID sloupec s PK (Primary Key) musí být označen jako `@Id` a musí nějakým způsobem generovat hodnoty, protože nechceš při vytváření uživatele přiřazovat ID sám. Důvod je ten, že když 2 uživatelé se budou registrovat zároveň, databáze jim přiřadí stejné ID, protože posílají požadavek ve stejnou chvíli, ale jednomu to spadne s chybou. Takže používáme generování pomocí nějaké sekvence nebo využijeme toho, co MySQL umí (a například PostgreSQL ne). To je AUTOINCREMENT. 

Anotace tedy bude `@GeneratedValue(strategy = GenerationType.IDENTITY)`. 
Samotná anotace je `@GeneratedValue` a jako parametr bere `strategy` s hodnotou `GenerationType.IDENTITY`. GenerationType je nějaký výčtový typ (enum), což je myslím i  v C#.